networkx/algorithms/centrality/tests/test_voterank.py

```diff
@@ -28,20 +28,54 @@ def test_voterank_centrality_1(self):
         )
         assert [0, 7, 6] == nx.voterank(G)
 
+    def test_vote_rank_centrality_1(self):
```

Comment:
It looks like these tests have the same name as the version they mimic (they don't --- but it looks like they do). That's one reason to make the names of the function different.